docs/reference/runtime-configuration.ipynb

```diff
@@ -39,173 +28,143 @@
+    "See below for the default file.\n",
     "\n",
-    "### `config.colors`:\n",
+    "The configuration is read from two files (if they exist).\n",
     "\n",
-    "The color scheme for the `scipp.table` and `scipp.show` functions, and the control buttons around the plots.\n",
+    "### User Configuration\n",
     "\n",
-    "- `attrs`: Color for attributes.\n",
-    "    - Default: `\"#ff5555\"`.\n",
-    "    - Allowed: A html color code or name.\n",
+    "A file called `config.yaml` in an operating-system-dependent location, specifically\n",
     "\n",
+    "- MacOS: `~/.config/scipp` or `~/Library/Application Support/scipp`\n",
+    "- Other Unix: `~/.config/scipp` or `/etc/scipp`\n",
+    "- Windows: `%APPDATA%\\scipp` where the `APPDATA` environment variable falls back to `%HOME%\\AppData\\Roaming` if undefined\n",
     "\n",
-    "- `coords`: Color for coordinates.\n",
-    "    - Default: `\"#c6e590\"`.\n",
-    "    - Allowed: A html color code or name.\n",
+    "Use\n",
+    "```python\n",
+    "sc.config.config_dir()\n",
+    "```\n",
+    "to find the folder on your system.\n",
+    "Or use\n",
+    "```python\n",
+    "sc.config.config_path()\n",
+    "```\n",
+    "to get the full path to the configuration file.\n",
     "\n",
+    "Note that these functions create the folder if it does not already exist.\n",
+    "But they do not create the file, you have to do that yourself.\n",
     "\n",
-    "- `data`: Color for data.\n",
-    "    - Default: `\"#f6d028\"`.\n",
-    "    - Allowed: A html color code or name.\n",
+    "### Working Directory\n",
     "\n",
+    "A file called `scipp.config.yaml` in the current working directory.\n",
+    "That is, the directory of the Jupyter notebook or from where you started the Python iterpreter.\n",
     "\n",
-    "- `masks`: Color for masks.\n",
-    "    - Default: `\"#c8c8c8\"`.\n",
-    "    - Allowed: A html color code or name.\n",
+    "If this file exists in the working directory, it takes precedence over the user configuration.\n",
     "\n",
+    "### Creating a Configuration File\n",
     "\n",
-    "- `hover`: Color for hovering on table rows.\n",
-    "    - Default: `\"#d6eaf8\"`.\n",
-    "    - Allowed: A html color code or name.\n",
+    "In order to use configuration files, copy the default configuration at the bottom of thise page to the location of your choice (See [User Configuration](#User-Configuration) and [Working Directory](#Working-Directory)) and modify any values you like.\n",
     "\n",
+    "Alternatively, you can provide only those values that you want to change and leave out everything else.\n",
+    "This way, scipp falls back to the default values.\n",
+    "Or, in the case of a file in the working directory, it falls back to the user configuration."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "    \n",
+    "**Warning**\n",
```

Comment:
I think nbsphinx is very picky about the warning formatting, I think this is a working example: https://github.com/scipp/scipp/blob/d21aa78ad1f8e8d7c2e191a7ce06ae77b81d6d52/docs/user-guide/binned-data/computation.ipynb?short_path=0f901aa#L269-L277